proparse/src/main/java/org/prorefactor/proparse/antlr4/Lexer.java

```diff
@@ -452,11 +469,13 @@ ProToken quotedString() throws IOException {
       }
     } // currChar==':'
 
-    return makeToken(ProParserTokenTypes.QSTRING);
+    return makeToken(PreprocessorParser.QSTRING);
   }
 
   ProToken digitStart() throws IOException {
-    int ttype = ProParserTokenTypes.NUMBER;
+    LOGGER.trace("Entering digitStart()");
+
+    int ttype = PreprocessorParser.NUMBER;
     for_loop : for (;;) {
       switch (currChar) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reduce the number of switch cases from 47 to at most 30. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1479)